crates/biome_analyze/src/rule.rs

```diff
@@ -313,6 +315,80 @@ impl RuleSourceKind {
     }
 }
 
+/// Rule domains
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
```

Comment:
I could derive `Copy` because this is a cheap and convenient.
```suggestion
#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
```